sensor_msgs/Image img
int64[] matrix